src/components/carrossel-oficinas/index.js

```diff
@@ -5,36 +5,44 @@ import 'swiper/css/navigation'
 import { Navigation } from 'swiper/modules'
 import { useState, useEffect } from 'react'
 import env from 'react-dotenv'
+import VerMais from './styled'
 import { cms } from '../../service/client'
-import ModalWorkshops from '../modal-oficinas'
+import { Link } from 'react-router-dom'
 
 const WorkshopsCarrossel = () => {
   const [attributes, setAttributes] = useState([])
   const urlCms = env.URL_CMS
 
   useEffect(() => {
-    cms.get('api/workshops/?populate=foto_divulgacao').then((response) => {
-      const { data } = response.data
-      const workshops = data.map((data) => {
-        return data.attributes
+    cms
+      .get('api/workshops/?populate=foto_divulgacao')
+      .then((response) => {
+        const { data } = response.data
+        const workshops = data.map((data) => {
+          if (data) {
+            return {
+              id: data.id,
+              name: data.attributes.nome,
+              imageUrl: data.attributes.foto_divulgacao.data[0].attributes.url
+            }
+          }
+        })
+        const workshopsSortedByName = workshops.sort((a, b) =>
```

Comment:
O que acontece se um workshop for `null ` ou `undefined`? De acordo com a linha 22, isso é possível. Acho que seria interessante colocar um filter antes disso tudo, vou sugerir isso em um comentário separado.

Reply:
https://github.com/aceleradora-TW/nossa-casa-front/pull/81/files#r1370488836